src/main-client.ts

```diff
@@ -194,19 +242,150 @@ export class MainClient extends BaseRestClient {
    *
   **/
 
-  createVirtualSubAccount(params: VirtualSubaccountParams): Promise<any> {
+  createSubAccount(params: CreateSubAccountParams): Promise<VirtualSubAccount> {
```

Comment:
```suggestion
  createVirtualSubAccount(params: CreateSubAccountParams): Promise<VirtualSubAccount> {
```

This is a breaking change (renaming an existing method that was here before the PR), can you change the method back to `createVirtualSubAccount`?